### PR TITLE
feat: sidebar item style improvement

### PR DESCRIPTION
**Pull request by Lythium4848**

### CODE REVIEW

File: lua/pixelui/elements/cl_sidebar.lua [hunk 1]
```diff
@@ -36,22 +36,28 @@ end
 function PANEL:Paint(w, h)
     local textCol = PIXEL.Colors.SecondaryText
     local backgroundCol = PIXEL.Colors.Transparent
+    local backgroundThingCol = ARSENIC.Colors.Transparent
```
Comment (TomDotBat) [inline lua/pixelui/elements/cl_sidebar.lua:39]: Use a better variable name.

Comment (TomDotBat) [reply lua/pixelui/elements/cl_sidebar.lua:39]: Invalid global name.

**Review by TomDotBat — CHANGES_REQUESTED** [review]

File: lua/pixelui/elements/cl_sidebar.lua (hunk 1 above)
Comment (Lythium4848) [reply lua/pixelui/elements/cl_sidebar.lua:39]: bitch what do you want me to call it

File: lua/pixelui/elements/cl_sidebar.lua [hunk 2]
```diff
@@ -36,18 +36,18 @@ end
 function PANEL:Paint(w, h)
     local textCol = PIXEL.Colors.SecondaryText
     local backgroundCol = PIXEL.Colors.Transparent
-    local backgroundThingCol = ARSENIC.Colors.Transparent
+    local backgroundStyleCol = PIXEL.Colors.Transparent
```
Comment (TomDotBat) [inline lua/pixelui/elements/cl_sidebar.lua:39]: Variable name is still misleading. You're drawing a line on the left of the button, that's not a background.

Comment (Lythium4848) [reply lua/pixelui/elements/cl_sidebar.lua:39]: AEFGY UASER ITY*F SRFG

**Comment by Lythium4848** [comment]
accept it alread

**Review by LucaTanks — CHANGES_REQUESTED** [review]
It's shit

**Comment by Lythium4848** [comment]
fuck off luca

**Comment by LucaTanks** [comment]
> fuck off luca

didnt ask

File: lua/pixelui/elements/cl_sidebar.lua (hunk 2 above)
Comment (LucaTanks) [reply lua/pixelui/elements/cl_sidebar.lua]: i agree

File: lua/pixelui/elements/cl_sidebar.lua [hunk 3]
```diff
@@ -36,22 +36,28 @@ end
 function PANEL:Paint(w, h)
     local textCol = PIXEL.Colors.SecondaryText
     local backgroundCol = PIXEL.Colors.Transparent
+    local hoverStyleCol = PIXEL.Colors.Transparent
```
Comment (TomDotBat) [inline lua/pixelui/elements/cl_sidebar.lua:39]: I don't understand your confusion with naming this variable. It's the colour of a line, `lineCol`?

Comment (Lythium4848) [reply lua/pixelui/elements/cl_sidebar.lua:39]: omfg

**Comment by LucaTanks** [comment]
Tom: `lineCol`
Lythium: *Decides to change it still*

**Comment by Lythium4848** [comment]
@LucaTanks it only fucking shows when it's hovered/selected so stfu

**Review by LucaTanks — APPROVED** [review]

**Pull request closed without merging**